src/pages/courses/index.vue

```diff
@@ -41,7 +45,11 @@ const rolesCanCreateCourse = [UserRole.Admin, UserRole.Teacher];
             </thead>
             <tbody>
               <tr v-for="{ course, teacher } in courses" :key="course" class="hover">
-                <td>
+                <td
+                  :class="{
+                    'min-w-[10rem] max-w-[12rem] whitespace-pre-wrap': !isDesktop,
```

Comment:
I'm curious whether `isDesktop` is a common usage in RWD? The last time I applied RWD, I was using like `lgAndUp`, `mdAndDown`. just discussion :D

Reply:
I think there are multiple methods to apply RWD. Like inline class with `lg`, `md`, for example.
Using `isDesktop` to separate two width ranges with dynamic binding is easier to maintain in my opinion.
If there's more than 2 device widths to consider. I think the one you mentioned would be a better solution.
I don't think we are making more design other than mobile right now.